API/Controllers/PointController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Core.DTO.PointDTO;
+using Core.Interfaces.CustomService;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace API.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PointController : ControllerBase
```

Comment:
This controller is responsible for a separate addition of a Point, but since we have in the offer combined Offer and Point. It will be needed to create a route.